src/gba_fixed.rs

```diff
@@ -1,23 +1,10 @@
-use core::ops::*;
-
-/// `i16` with 8 bits of fixed-point fraction.
-///
-/// This is used by the affine matrix entries.
-#[allow(non_camel_case_types)]
-pub type i16fx8 = Fixed<i16, 8>;
+//! Basic fixed point math module used as a fallback if the `fixed` crate isn't
```

Comment:
I know I mentioned it before, but I'd like to see all of this `track_caller`-ed to hell and back. Extra important given that the default GBA targets don't support unwinding so the only panic location you get is the one the compiler gives you at the panic site.

Reply:
Also, `bytemuck` integration would be lovely. `num_traits` too, if at all possible. I've ended up writing my own fixed-point type because sadly these types don't fill these gaps.